dtfs-central-api/src/services/portal/deal.service.add-deal-cancelled-activity.test.ts

```diff
@@ -0,0 +1,74 @@
+import { Activity, AnyObject, Deal, DEAL_TYPE, PORTAL_ACTIVITY_LABEL, PORTAL_ACTIVITY_TYPE } from '@ukef/dtfs2-common';
+import { generateSystemAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { getUnixTime } from 'date-fns';
+import { PortalDealService } from './deal.service';
+
+const updateDealMock = jest.fn() as jest.Mock<Deal>;
+
+jest.mock('../../v1/controllers/portal/gef-deal/update-deal', () => ({
+  updateDeal: (params: AnyObject) => updateDealMock(params),
+}));
+
+const dealId = 'dealId';
+
+const portalActivities: Array<Activity> = [];
```

Comment:
As commented elsewhere - could you update this to `Activity[]` (and similar anywhere else)